JakinChan200/CS180_Roald
# Add CORS for Demo

## Overview
Quick fix to add CORS to backend, so that requests work on CORS-requiring browsers (Chrome, Safari)
Removes node_modules from server

## Materials
- N/A


## Pictures
N/A

## Manual Testing Guide
- Check the message on the home page is different before and after running the server

Pull requests should receive approval before merge.
